src/NovelRT/CMakeLists.txt

```diff
@@ -184,6 +183,9 @@ if(WIN32)
   add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy
     $<TARGET_FILE:tbb>
     $<TARGET_FILE_DIR:Engine>/$<TARGET_FILE_NAME:tbb>)
+  add_custom_command(TARGET Engine POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy
+    $<TARGET_FILE:zlib>
+    $<TARGET_FILE_DIR:Engine>/$<TARGET_FILE_NAME:zlib>)
```

Comment:
Do we do this for other targets? Or how are they handled?

Reply:
We do this for every target that builds a shared library - with that said, this can be removed since it's static now